From ce03dcb3a2f971980e9508b0e2c56a004320fc37 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Javier=20Jard=C3=B3n?= <jjardon@gnome.org>
Date: Mon, 28 Aug 2017 23:14:51 +0100
Subject: [PATCH] build: Add --disable-valgrind argument to configure

This can be useful for systems without valgrind installed, like continuous.
---
 configure | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 903f82b63..81e0d2797 100755
--- a/configure.ac
+++ b/configure.ac
@@ -80,6 +80,7 @@ while (($# > 0)); do
 	--disable-introspection) enable_introspection='-Ddisable_introspection=true';;
 	--disable-fwupd) enable_fwupd='-Denable-fwupd=false';;
 	--disable-packagekit) enable_packagekit='-Denable-packagekit=false';;
+	--disable-valgrind) enable_valgrind='-Denable-valgrind=false';;
 	*) echo -e "\e[1;33mINFO\e[0m: Ignoring unknown option '$1'";;
     esac
     shift
@@ -125,7 +126,7 @@ echo "  libdir:...... ${libdir}"
 echo "  mandir:...... ${mandir}"
 echo "  includedir:.. ${includedir}"
 echo "  additional:.."
-echo "    - ${enable_docs} ${enable_man} ${enable_introspection} ${enable_fwupd} ${enable_packagekit}"
+echo "    - ${enable_docs} ${enable_man} ${enable_introspection} ${enable_fwupd} ${enable_packagekit} ${enable_valgrind}"
 
 exec ${MESON} \
 	--prefix=${prefix} \
@@ -142,6 +143,7 @@ exec ${MESON} \
 	${enable_introspection} \
 	${enable_fwupd} \
 	${enable_packagekit} \
+	${enable_valgrind} \
 	${srcdir}
 
 # vim: ai ts=8 noet sts=2 ft=sh
-- 
GitLab

